extensions/lifecycle/bucketProcessor/LifecycleBucketProcessor.js

```diff
@@ -76,6 +77,17 @@ class LifecycleBucketProcessor {
         this._producer = null;
         this._kafkaBacklogMetrics = null;
 
+        this._supportedRulesObject = {};
+        supportedLifecycleRules.forEach(rule => {
+            this._supportedRulesObject[rule] = { enabled: true };
+        });
+        this._hasSupportedRules = Array.isArray(supportedLifecycleRules) &&
```

Comment:
Wouldn't we need to also check that at least one of the supported rules has the `enabled: true` flag? Just to maintain the same behavior than before.

Reply:
I don't believe so. the `supportedLifecycleRules` will be a constant from arsenal, and the generated object will always have `enabled: true` for each key.